testcontainers/build.gradle

```diff
@@ -0,0 +1,92 @@
+plugins {
+    id 'java'
+}
+
+repositories {
+    mavenCentral()
+}
+
+test {
+    testLogging {
+        events "passed", "skipped", "failed"
+    }
+}
+
+dependencies {
+
+    testImplementation project(':integration-tests').sourceSets.test.output
+
+    testImplementation(project(':registry-common')) { transitive = false }
+    testImplementation(project(':registry-common-client')) { transitive = false }
+    testImplementation(project(':common-auth')) { transitive = false }
+
+    testImplementation(project(':schema-registry:schema-registry-core')) { transitive = false }
+    testImplementation(project(':schema-registry:schema-registry-client')) { transitive = false }
+    testImplementation(project(':schema-registry:schema-registry-serdes')) { transitive = false }
+    testImplementation(project(':schema-registry:schema-registry-common')) { transitive = false }
+    testImplementation(project(':schema-registry:schema-registry-authorizer')) { transitive = false }
+    testImplementation(project(':schema-registry:schema-registry-webservice')) { transitive = false }
+    testImplementation(project(path: ':schema-registry:jersey-shaded', configuration: 'shadow')) { transitive = false }
+    testImplementation(project(':schema-registry:schema-registry-client')) { transitive = false }
+    testImplementation(project(':schema-registry:oauth2-common')) { transitive = false }
+    testImplementation(project(':schema-registry:oauth2-authenticator')) { transitive = false }
+    testImplementation(project(':schema-registry:ssl-authenticator')) { transitive = false }
+
+    testImplementation(project(':atlas-integration:atlas-module')) { transitive = false }
+    testImplementation(project(':atlas-integration:atlas-common')) { transitive = false }
+    testImplementation(project(':atlas-integration:atlas-module')) { transitive = false }
+
+    testImplementation(project(':storage:storage-tool')) { transitive = false }
+    testImplementation(project(':storage:storage-core')) { transitive = false }
+    testImplementation(project(':storage:storage-common')) { transitive = false }
+
+    testImplementation libraries.lombok
+    testAnnotationProcessor libraries.lombok
+
+    testImplementation('com.cloudera.dim.kafka-connect:tests:0.0.1.7.2.17.0-SNAPSHOT')
```

Comment:
This version is not reachable, it breaks the compilation.

Reply:
More precisely, the whole library is unavailable

Reply:
Thanks, fixed.